src/App.tsx

```diff
@@ -35,7 +47,13 @@ function RNGalleryScreenWrapper({navigation}) {
   const Component = RNGalleryList[state.index].component;
   return (
     <View style={styles.container}>
-      <Button title="Menu" onPress={() => navigation.openDrawer()} />
+      <TouchableHighlight
+        style={styles.menu}
+        onPress={() => navigation.openDrawer()}
+        activeOpacity={0.5783}
+        underlayColor="rgba(0, 0, 0, 0.0241);">
```

Comment:
Similar question: Where does this magic opacity number come from?

Reply:
Sync'd offline